bot/exts/fun/snakes/_utils.py

```diff
@@ -355,7 +355,7 @@ def frame_to_png_bytes(image: Image) -> io.BytesIO:
 
 
 log = get_logger(__name__)
-START_EMOJI = "\u2611"     # :ballot_box_with_check: - Start the game
+START_EMOJI = Emojis.check     # :ballot_box_with_check: - Start the game
 CANCEL_EMOJI = "\u274C"    # :x: - Cancel or leave the game
```

Comment:
I'm fine either way, but why not remove the `:*: - ` hint in the comments (previously used for the much less descriptive escape sequences) since they're now determined by the settings of the constants?